CONTRIBUTING.md

```diff
@@ -1,256 +1,339 @@
+
+### Remote packages
+
+_Remote_ packages does not include any packaged software directly in the nupkg archive, instead it downloads what is necessary when a user installs the package.
+This is the most common type seen on the Chocolatey Community Repository, and is the only valid approach if the software being packaged does not allow for it to be redistributed and the software authors are not granting and redistribution rights.
+Additionally, when a package becomes too large to be uploaded to Chocolatey Community Repository it is valid to change an embedded package to a remote package.
+
+### Metadata
 
-Keep tags **lowercase**. Use `-` between words. The following tags are mandatory to use if they apply:
+This section details the conventions and guideliens that are related to changing or updating metadata for each package.
+Metadata are explicitly only the information that is defined for the package in its related `.nuspec` file.
 
-|tag|meaning|
-|---|---|
-| `foss` | for all free and open source packages|
-| `cross-platform` | for all packages available on other platforms then Windows|
-| `freeware` | for all freeware but not open source packages|
-| `trial` | for all propriatery commercial tools that require license|
-| `cli` | for all command line tools|
-| `games` | for all games |
+#### Naming
 
-**Note**: If the software provides both free and paid variant within the same executable so that free version doesn't have a time limit, mark them as both `freeware` and `trial`.
+For automatic packages the name of the package is taken from its root directory name, if the original idenifier of the existing package is not in lowercase this needs to be overridden in an `update.ps1` script.
+For manual packages this is defined directly in the package nuspec file, in general this should also be in lowercase but for existing packages it should use the same casing as the original identifier.
+This root directory name should always be in lower case and follow the official Chocolatey naming conventions: https://docs.chocolatey.org/en-us/create/create-packages#naming-your-package.
 
-### 1.2.3 Informative description
+#### Dependency versions
 
-Description is maintained in the `README.md` file in the root of the package and automatically set in the nuspec by AU during an update. It should include the following **2nd level headers (##) in the given order**:
+1. When taking a dependency on an extension, a minimum version of that dependency must be specified. Without this, any version satisfies the dependency. That means unless someone upgrades the extension outside of their process or incidentally install some package that uses newer version explictly set, it will not automatically upgrade to the latest version. Changes to a package that does not specify a minimum version, or exact version is will not be accepted.
```

Comment:
```suggestion
1. When taking a dependency on an extension, a minimum version of that dependency must be specified. Without this, any version satisfies the dependency. That means unless someone upgrades the extension outside of their process or incidentally install some package that uses an explicitly set newer version, it will not automatically upgrade to the latest version. Changes to a package that do not specify a minimum version, or exact version, not be accepted.
```